hwilib/devices/bitbox02.py

```diff
@@ -135,6 +133,40 @@ def attestation_check(self, result: bool) -> None:
             )
 
 
+def _parse_path(nstr: str) -> Sequence[int]:
+    """
+    Adapted from trezorlib.tools.parse_path.
+    Convert BIP32 path string to list of uint32 integers with hardened flags.
+    Several conventions are supported to set the hardened flag: -1, 1', 1h
```

Comment:
just curious: `-1` who uses this?

Reply:
I was also surprised to see that :shrug: